### ISSUE TITLE
Curatorial Narrative field updates

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vatican_exhibits#270]
The Curatorial Narrative field works well but there are some improvements to the way it is displayed that I think would be good to fix before the BAV curators start to work with it:

- Make the input field a text area, since they are likely to put somewhat lengthy text in it
- The field is duplicated, on the Configuration > Metadata page:

<img width="365" alt="configuration_-_metadata___greek_paleography_-_thematic_pathways_of_medieval_manuscripts" src="https://user-images.githubusercontent.com/101482/42912886-a1ed6ea0-8aa6-11e8-8e57-c20270c8dc73.png">

 and so also on the manuscript show page when editing (I only entered text in the second input but it showed up in the other version of the field the next time I edited the page):

<img width="603" alt="project_overview_-_thematic_pathways_of_medieval_manuscripts" src="https://user-images.githubusercontent.com/101482/42912889-a69aaf94-8aa6-11e8-8cd1-adc7713c575c.png">

I guess we can just delete the extra version of the field on the Configuration > Metadata page but it seems better to give the BAV code that doesn't require them to do that?

Similarly, although less important, it would be nice if the default label was "Curatorial Narrative" instead of "Curatorial Narrative Tesim" although I realize the display label can be edited. It's just an extra thing we have to remember to explain or do for them.

**Comment by mejackreed** [comment]
@ggeisler #284 removes the duplication each time `rake db:seed` is run, but we can just manually delete the extra fields from the MD config for each exhibit. After #284 this shouldn't keep happening.

**Comment by ggeisler** [comment]
I'm not creating a new ticket for this, because I'm not sure if all will be fine if we completely start over, but want to get confirmation that it will be fine, despite what I found below where I no longer have a curatorial narrative input field when creating a new exhibit:

When I create a new exhibit (on http://vatican-dev.sul.stanford.edu), I see the `Curatorial Narrative Tesim` field in the field list on the Configuration > Metadata page, so that’s good. But when I edit a manuscript item, there is no corresponding input for me to add the narrative text, just the Tags input.

My new test exhibit showing the lack of input field is here: http://vatican-dev.sul.stanford.edu/cn-test/catalog/Arch_Cap_S_Pietro_D_182/edit

**Comment by mejackreed** [comment]
I think the issue here is that our `seed.rb` file is the thing that sets this up manually for each exhibit. So new exhibits would need to go in and manually create this field.

**Comment by ggeisler** [comment]
Okay sounds good, thanks. Just wanted to get your confirmation before remove this from our must-have list.
